### PR TITLE
Allow for the request of archived groups

### PR DESCRIPTION
**Pull request by WilliamBZA**
Adds the ability to request the groups of archived messages.
ServicePulse uses this to display the information. If ServicePulse connects to a version of ServiceControl that does not support this then the ServicePulse behaviour does not change.

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Ping @ramonsmits and @WojcikMike

**Comment by danielmarbach** [comment]
I personally would like to see an acceptance test that covers those new APIs. We have plenty of acceptance tests in Recoverability and group that also test the archival that should be extended or new tests added to verify future changes to the product don't break the functionality

File: src/ServiceControl/Infrastructure/WebApi/RootController.cs [hunk 1]
```diff
@@ -39,7 +39,8 @@ public OkNegotiatedContentResult<RootUrls> Urls()
                 LicenseStatus = license.IsValid ? "valid" : "invalid",
                 LicenseDetails = baseUrl + "license",
                 Configuration = baseUrl + "configuration",
-                EventLogItems = baseUrl + "eventlogitems"
+                EventLogItems = baseUrl + "eventlogitems",
+                ArchivedGroupsUrl = baseUrl + "errors/groups/{classifier?}"
```
Comment (danielmarbach) [inline src/ServiceControl/Infrastructure/WebApi/RootController.cs:43]: Are we deliberately not adding `archive/groups/id/{groupId}` here or did I miss that?

File: src/ServiceControl.AcceptanceTests/Recoverability/Groups/When_a_group_is_archived.cs [hunk 2]
```diff
@@ -62,6 +62,59 @@ await bus.SendLocal<MyMessage>(m => m.MessageNumber = 2)
                 .Run();
         }
 
+        [Test]
+        public async Task All_archived_messages_should_be_grouped()
```
Comment (danielmarbach) [inline src/ServiceControl.AcceptanceTests/Recoverability/Groups/When_a_group_is_archived.cs:66]: Looks good. Should we have another test that also exercises the `archive/groups/id/{groupId}` route?

Comment (WilliamBZA) [reply src/ServiceControl.AcceptanceTests/Recoverability/Groups/When_a_group_is_archived.cs:66]: Added

File: src/ServiceControl/Infrastructure/WebApi/RootController.cs (hunk 1 above)
Comment (WilliamBZA) [reply src/ServiceControl/Infrastructure/WebApi/RootController.cs]: Nope, added

**Pull request merged**